woocommerce/woocommerce
# Monorepo: Update Build and e2e Github Actions

### Changes proposed in this Pull Request:

Update paths in Github workflows for build and e2e testing. Other changes were required to get this working.

* Add back in `lerna`, which was removed prematurely.
* Add back `.eslintrc.js` for the woocommerce plugin. It will be removed in favour of a top level eslintrc, but that is out of scope for this PR and the initial monorepo restructure.
* WooCommerce's Build Action required a change to match the new structure. The PR for corresponding changes is [here](https://github.com/woocommerce/action-build/pull/3).

### How to test the changes in this Pull Request:

See the `Build zip for PR / Runs E2E tests. (pull_request)` Github action pass

![image](https://user-images.githubusercontent.com/1922453/137055991-4787b1d4-58a3-406e-a296-4b581396da11.png)
